src/components/Sidebar/Sidebar.less

```diff
@@ -0,0 +1,213 @@
+@size-Sidebar-icon: 20px;
```

Comment:
Also, you should use the `@size-expander-button: 20px;` variable for this one since it already exists.